environment.yml

```diff
@@ -6,6 +6,7 @@ channels:
 dependencies:
   - python=3.7
   - pip
+  - pyirf>=0.2.0
```

Comment:
I think there will be a release soon and this should be changed to the latest version

Reply:
And requirung == probably makes sense. Or at least `!=0.x.0` as there will probably be breaking changes down the road until we have a more stable interface.

Reply:
Change to `~=0.3.0`, we made the release today.